googleapis/google-cloud-node
# migrate code from googleapis/nodejs-apigee-connect

- feat: initial stub of library
- feat: initial samples and tests
- feat: add initial samples and tests (#5)
- build: add auto-approve to Node libraries (#1100) (#7)
- chore(deps): update dependency sinon to v11 (#3)
- chore: release 1.0.0 (#1)
- build(nodejs): name should be @google-cloud/apigee-connect (#12)
- fix(deps): require google-gax v2.17.0 (#13)
- chore: release 1.0.1 (#15)
- fix(deps): google-gax v2.17.1 (#16)
- chore: release 1.0.2 (#17)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#18)
- chore: release 1.0.3 (#19)
- chore: release 1.0.3 (#20)
- chore(nodejs): update client ref docs link in metadata (#26)
- fix(build): migrate to using main branch (#28)
- chore: release 1.0.4 (#29)
- fix(deps): google-gax v2.24.1 (#30)
- chore: release 1.0.5 (#31)
- feat: turns on self-signed JWT feature flag (#32)
- chore: release 1.1.0 (#33)
- fix: flag the module as GA (#34)
- chore: release 1.1.1 (#35)
- build(node): update deps used during postprocessing (#1243) (#43)
- chore(cloud-rad): delete api-extractor config (#45)
- chore: fix the wrong post processor image (#47)
- chore(deps): update dependency @types/node to v16 (#46)
- chore(deps): update dependency sinon to v12 (#48)
- docs(samples): add example tags to generated samples (#49)
- build: add generated samples to .eslintignore (#50)
- docs(node): support "stable"/"preview" release level (#1312) (#53)
- chore: add api_shortname and library_type to repo metadata (#52)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#55)
- test(nodejs): remove 15 add 16 (#1322) (#57)
- build: update copyright year to 2022 (#59)
- chore: update v2.12.0 gapic-generator-typescript (#61)
- chore(deps): update dependency sinon to v13 (#63)
- docs(nodejs): version support policy edits (#1346) (#65)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#66)
- build(node): update client library version in samples metadata (#1356) (#76)
- chore(deps): update dependency @types/mocha to v9 (#77)
- chore(deps): update dependency sinon to v14 (#80)
- build!: update library to use Node 12 (#81)
- fix: fixes for dynamic routing and streaming descriptors (#84)
- chore(main): release 2.0.0 (#83)
- chore(deps): update dependency jsdoc-fresh to v2 (#86)
- chore(deps): update dependency jsdoc-region-tag to v2 (#87)
- feat: support regapic LRO (#88)
- chore(main): release 2.1.0 (#89)
- chore(deps): update dependency linkinator to v4 (#93)
- fix: better support for fallback mode (#94)
- fix: change import long to require (#95)
- chore(main): release 2.1.1 (#96)
- fix: do not import the whole google-gax from proto JS (#1553) (#98)
- fix: allow passing gax instance to client constructor (#99)
- chore(main): release 2.1.2 (#100)
- fix: preserve default values in x-goog-request-params header (#101)
- test: use fully qualified request type name in tests (#104)
- fix(deps): use google-gax v3.5.2 (#108)
- chore(deps): update dependency @types/node to v18 (#106)
- fix: regenerated protos JS and TS definitions (#111)
- chore(deps): update dependency jsdoc to v4 (#110)
- chore(main): release 2.1.3 (#102)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
